mesheryctl/internal/cli/root/perf/perf.go

```diff
@@ -69,7 +69,7 @@ mesheryctl perf result -o yaml
 			return cmd.Help()
 		}
 		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
-			return errors.New(utils.PerfError(fmt.Sprintf("'%s' is a invalid command.  Use 'mesheryctl perf --help' to display usage guide.'\n", args[0])))
+			return errors.New(utils.PerfError(fmt.Sprintf("You are misspelling the command as '%s'. Use 'mesheryctl perf --help' to display usage guide.'\n", args[0])))
```

Comment:
This UX seems interesting... Could we implement for other commands too if such line exists?

Reply:
But wait... it seems it's not applicable to all use cases... we might need to reconsider this code line

Reply:
> But wait... it seems it's not applicable to all use cases coz even if user provides a "good" wrong subcommand it'll show as misspelt... we might need to reconsider this code line IMO

Yes, I have thought about this case but I have felt that they would see the user guide, which will be provided in the error message to get to know about the commands if they have used some invalid command. So,  what do you suggest I do?